themes/bootstrap5/templates/RecordDriver/DefaultRecord/core.phtml

```diff
@@ -65,14 +66,15 @@
     <?php endif; ?>
 
     <?php if ($this->searchOptions($this->driver->getSourceIdentifier())->getVersionsAction()): ?>
-      <?=$this->record($this->driver)->renderTemplate('versions-link.phtml')?>
+      <?=$recordHelper->renderTemplate('versions-link.phtml')?>
     <?php endif; ?>
 
     <?php $summary = $this->driver->getSummary(); ?>
-    <?php if ($summary = isset($summary[0]) ? $this->escapeHtml($summary[0]) : false): ?>
-      <p><?=$this->truncate($summary, 300)?></p>
+    <?php if ($summary = $summary[0] ?? null): ?>
+      <?php $shortSummary = $this->truncate($summary, 300); ?>
```

Comment:
Do we need to worry about truncate splitting HTML and causing broken markup? I thought we addressed this somewhere/somehow, but in my quick review just now I don't see where.

Reply:
No, we shouldn't need to. Truncate follows your brilliant idea that if the plain string exceeds the maximum length, only it is returned (truncated). I added a comment about this to Truncate helper.